packages/shared-data/pricing.ts

```diff
@@ -192,6 +193,17 @@ export const pricing: Pricing = {
         },
         usage_based: true,
       },
+      {
+        key: 'database.egress',
```

Comment:
Wondering whether it would make sense to have a more specific tooltip text, i.e. a different `key`

Reply:
yeah wanted to go through pricing page + tooltips still

Reply:
Yep